lib/api/src/index.tsx

```diff
@@ -389,14 +389,25 @@ export function useAddonState<S>(addonId: string, defaultState?: S) {
       ...stateChangeHandlers,
       ...stateInitializationHandlers,
     };
-  }, [addonId]);
+  }, [stateId]);
 
   const emit = useChannel(allListeners);
   return [
     state,
     (newStateOrMerger: S | StateMerger<S>, options?: Options) => {
       setState(newStateOrMerger, options);
-      emit(`${ADDON_STATE_CHANGED}-manager-${addonId}`, newStateOrMerger);
+      emit(`${SHARED_STATE_CHANGED}-manager-${stateId}`, newStateOrMerger);
     },
   ] as [S, (newStateOrMerger: S | StateMerger<S>, options?: Options) => void];
 }
+
+export function useAddonState<S>(addonId: string, defaultState?: S) {
+  return useSharedState<S>(addonId, defaultState);
+}
+
+export function useStoryState<S>(defaultState?: S) {
+  const {
+    state: { storyId },
+  } = useContext(ManagerContext);
```

Comment:
Can use `useStorybookState` ?

Reply:
sure, just a sec

Reply:
oops - spoken too early. There is already a useStorybookState :

https://github.com/storybookjs/storybook/blob/c35919c497ec03c8d20ec3c074a974ec04876142/lib/api/src/index.tsx#L300